infra/terraform/aks/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "resource_group_name" {
+  default = "ecosystem-ci"
+}
+
+variable "location" {
+  default = "Germany West Central"
```

Comment:
So ... this is what came naturaly, but didn't this location have problems with Redis Cache creation?

Reply:
Its not really a problem of the location.
Creating an azure cache deployment does take a considerable amount of time and according to [this feedback post](https://feedback.azure.com/forums/169382-cache/suggestions/7049852-work-on-creation-time-of-redis-cache) the mains reason is:
`"Azure Cache currently runs on Windows VM's and the creation time is largely determined by how fast the underlying VM's become available on-demand."`